client/consensus/slots/src/lib.rs

```diff
@@ -98,6 +101,9 @@ pub trait SimpleSlotWorker<B: BlockT> {
 	/// Epoch data necessary for authoring.
 	type EpochData: Send + 'static;
 
+	/// Strategy for deciding if and when to backoff authoring new blocks if finality starts to lag.
+	type BackoffAuthoringBlocksStrategy: BackoffAuthoringBlocksStrategy<NumberFor<B>>;
```

Comment:
Why does this needs to be an associated type here when it is not used in here?

Reply:
Ah good catch! Yeah I don't think this one is needed (anymore)